app/components/ui/Modals/AdminPanel/components/OwnerProduct.tsx

```diff
@@ -10,7 +10,7 @@ import { OwnerProductHeader } from "./OwnerProductHeader"
 
 type Props = IDBProduct
 
-function OwnerProduct({ ...ownerProduct }: Props) {
+export function OwnerProduct({ ...ownerProduct }: Props) {
```

Comment:
> Delete product form created without header

this commit way better prvious ones - keep going

only 1 little thing that this - you may don't include file in commit and do thin minor change in separate commit